cypress/e2e/settings/tenant/verify-item-status-language-when-locale-is-german.cy.js

```diff
@@ -0,0 +1,81 @@
+import Helper from '../../../support/fragments/finance/financeHelper';
+import { Permissions } from '../../../support/dictionary';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import TenantPane, { TENANTS } from '../../../support/fragments/settings/tenant/tenantPane';
+import Users from '../../../support/fragments/users/users';
+import { randomFourDigitNumber } from '../../../support/utils/stringTools';
+import SettingsMenu from '../../../support/fragments/settingsMenu';
+import { Localization } from '../../../support/fragments/settings/tenant/general';
+import TemporarySessionLocale from '../../../support/fragments/settings/developer/session-locale/temporarySessionLocale';
+import TopMenu from '../../../support/fragments/topMenu';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import InventoryInstance from '../../../support/fragments/inventory/inventoryInstance';
+
+describe('Settings: Tenant', () => {
+  const testData = {
+    buttonLanguage: 'German (Germany) / Deutsch (Deutschland)',
+    defaultButtonLanguage: 'American English / American English',
+    paneHeaderTitle: 'TEMPORÄRES Session-Gebietsschema',
+    defaultPaneHeaderTitle: 'TEMPORARY Session locale',
+  };
+  const instanceTitle = `C407670 autotestInstance ${randomFourDigitNumber()}`;
+  const itemBarcode = Helper.getRandomBarcode();
+  const HoldingData = {
+    rowNumber: 1,
+    status: 'Verfügbar',
+    accordionHeaderLanguageValue: 'Bestand: Main Library',
+  };
+
+  before('Create test data', () => {
+    cy.getAdminToken().then(() => {
+      InventoryInstances.createInstanceViaApi(instanceTitle, itemBarcode);
```

Comment:
The instance should has 1 holdings and 2 items

Reply:
As I can see in createInstanceViaApi method it indeed has 1 holding and 2 items that are created inside of an instance.
Also as test running I also see 2 items.
![image](https://github.com/folio-org/stripes-testing/assets/91249189/7a065c10-c6bc-4b0c-9781-b4d3c7ff0824)